docs/en/ingest-management/elastic-agent/install-standalone-elastic-agent.asciidoc

```diff
@@ -13,11 +13,18 @@ We recommend using <<install-fleet-managed-elastic-agent,{fleet}-managed {agent}
 when possible, because it makes the management and upgrade of your agents
 considerably easier. 
 
-IMPORTANT: Standalone agents are unable to upgrade to new integration package
+[IMPORTANT]
+====
+Note the following restrictions for running standalone {agent}:
+
+* Standalone agents are unable to upgrade to new integration package
 versions automatically. When you upgrade the integration in {kib}, you'll
 need to update the standalone policy manually.
 
-NOTE: You can install only a single {agent} per host.
+* The file path where {agent} is installed cannot exceed 103 characters.
```

Comment:
This is not just about the installation path, all paths, including data path for specific integration shouldn't exceed 103 characters.
Maybe @joshdover or @cmacknz have a better wording here.

Reply:
How about this:
> Elastic Agent file paths cannot exceed 103 characters. This restriction applies to all paths, including the Elastic Agent install directory and the data path used by any integrations.